src/Freya.Core/Builder.fs

```diff
@@ -74,21 +73,6 @@ type FreyaBuilder () =
             this.While (enum.MoveNext, this.Delay (fun () -> 
                 body enum.Current)))
 
+/// The instance of <see cref="FreyaBuilder" /> used for constructing
+/// <see cref="Freya{T}" /> computations.
 let freya = new FreyaBuilder ()
```

Comment:
I think the changes in here are all excellent - I've been hmmmm'ing over how that's structured for a while and this seems much better now.